backend/src/main/java/com/barmjz/productivityapp/user/UserService.java

```diff
@@ -31,6 +31,6 @@ public UserDetails loadUserByUsername(String email) throws UsernameNotFoundExcep
         return userRepo
```

Comment:
I recommend changing the name of method to "loadUserByEmail()"

Reply:
This method overrides a specific and necessary method in UserDetailsService class which is related to Spring Security and is implemented by UserService so changing the name would break the app as it's required to implement this method and override it with the same name.

Reply:
Okay, got it